apps/jubmoji-quest/src/pages/api/generateApplePass.ts

```diff
@@ -36,7 +31,9 @@ export default async function handler(
     });
   }
 
-  if (parseInt(number as string) > COLLECTION_SIZE) {
+  const serializedJubmojis = collection.toString().split(",");
+
+  if (serializedJubmojis.length > COLLECTION_SIZE) {
```

Comment:
What is the point of this check? Is there a reason we need to handle this edge case where a user sends us 500 Jubmojis to generate a pass with?